examples/commoncrawl/src/main/ml-config/security/users/exporter.json

```diff
@@ -0,0 +1,17 @@
+ 
```

Comment:
Do you know if this user is needed by any scripts? If not, let's delete it for now. If we need to keep it, we'll need to gitignore every directory under src/main/ml-config except this one.

Reply:
This user performs a perf test on nightly for CSV export

Reply:
So, no, I guess we technically don't need it for examples proj